src/Graphics/Canvas.js

```diff
@@ -593,3 +593,9 @@ export function bezierCurveTo(ctx) {
     };
   };
 }
+
+export function imageSmoothingEnabled(ctx) {
+  return function(value) {
+    ctx.imageSmoothingEnabled = value;
```

Comment:
I think you are missing a 
```
    return function() {
```
to make it return an `Effect`

Reply:
You're right! thanks.

Reply:
Ah, right. thanks.